src/content/reference/kernel-settings/_index.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Tuned kernel settings
+description: Complete list of the tuned CoreOS kernel settings for Giant Swarm clusters.
+date: 2020-02-20
+layout: subsection
+weight: 100
+---
+
+# Kernel settings
+
+To run applications on a Giant Swarm cluster properly, some kernel settings need to be tuned at the OS level.
```

Comment:
This sounds to me a bit as if there is something that needs to be done by the reader of the document. Maybe we can be more clear that this is done by us and documenting the current settings.